lm_survey/survey/question.py

```diff
@@ -44,10 +44,10 @@ def __init__(
     ) -> None:
         self.key = key
         self.text = text
-        self.invalid_options = set(invalid_options)
+        self.invalid_options = set(map(str.lower, (invalid_options)))
```

Comment:
Double check that when we're checking whether a key is valid/invalid we're lowercasing the other one too

Reply:
Take away lower, and instead we will change the data before we do anything with it